src/redux/actions/actionTypes.js

```diff
@@ -1,3 +1,3 @@
-export const LOAD_TODOS = 'LOAD_TODOS';
-export const LOAD_TODOS_SUCCESS = 'LOAD_TODOS_SUCCESS';
-export const LOAD_TODOS_ERROR = 'LOAD_TODOS_ERROR';
+export const LOAD_TODOS_REQUEST = 'LOAD_TODOS_START';
```

Comment:
wrong name mapping  why not consistent ?

Reply:
got trapped on IDE